source/module.c

```diff
@@ -804,6 +804,7 @@ static PyMethodDef s_module_methods[] = {
 
 static const char s_module_name[] = "_awscrt";
 PyDoc_STRVAR(s_module_doc, "C extension for binding AWS implementations of MQTT, HTTP, and friends");
+AWS_STATIC_STRING_FROM_LITERAL(s_crash_handler_env_var, "AWS_CRT_ENABLE_CRASH_HANDLER");
```

Comment:
debatable: shorter name, similar to pre-existing "AWS_CRT_MEMORY_TRACING"?
```suggestion
AWS_STATIC_STRING_FROM_LITERAL(s_crash_handler_env_var, "AWS_CRT_CRASH_HANDLER");
```